src/routes/+page.svelte

```diff
@@ -25,17 +26,85 @@
 								dynamic storytelling.
 							</h2>
 						</div>
-						<div class="mt-16 flex justify-end">
-							<Button class="bg-tertiary-400 w-1/2 rounded-none py-4 font-primary !text-4xl !text-quaternary lg:w-1/4" href="/try-for-free"
-								>TRY FOR FREE</Button
+						<div class="mt-16 flex justify-end ">
+							<Button class="bg-tertiary-400 w-1/2 rounded py-4 font-primary !text-4xl !text-quaternary lg:w-1/4" href="/try-for-free"
```

Comment:
I think it's better to use something like bg-tertiary-300 here. Or even lower. Makes it easier to read.